Dockerfile

```diff
@@ -21,11 +21,11 @@ ARG YARN_VERSION=v1.22.19
 
 # bootstrap yarn so we can install and run the other tools.
 RUN CACHED_YARN=./artifacts/yarn-${YARN_VERSION}.tar.gz; \
-    if [ -f ${CACHED_YARN} ]; then \
-      npm install ${CACHED_YARN}; \
-    else \
-      npm install https://github.com/yarnpkg/yarn/releases/download/${YARN_VERSION}/yarn-${YARN_VERSION}.tar.gz; \
-    fi
+  if [ -f ${CACHED_YARN} ]; then \
```

Comment:
Did you mean to reformat the Dockerfile?

Reply:
Not really, my IDE just autoformats on save. I can revert if needed.

Reply:
I find the previous indentation easier to read if you can temporarily disable that